### PR TITLE
Some re-organization of README.md and Tools.md

### PR DESCRIPTION
**Pull request by soupi**
[View it here](https://github.com/soupi/documentation).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,42 @@
...
+PureScript is a small strongly typed programming language that compiles to JavaScript.
+To get a better overview of PureScript, visit [The PureScript Website](http://purescript.org).
 
-- [Maintained Packages](ecosystem/Maintained-Packages.md)
-- [Style Guide](guides/Style-Guide.md)
-- [Related Projects](language/Related-Projects.md)
+## Getting Started
+
+- [Getting Started guide](http://www.purescript.org/learn/getting-started/): Download PureScript and build your first project
+- [Try PureScript](http://try.purescript.org): Try PureScript in your browser
```
Comment (hdgarrood) [inline README.md]: Perhaps PureScript by Example should go here too?

File: README.md [hunk 2]
```diff
@@ -62,10 +53,10 @@ The [PureScript Book](https://leanpub.com/purescript/read) is the recommended ap
...
 - [Differences from Haskell](language/Differences-from-Haskell.md)
-- [Alternate compiler backends](ecosystem/Alternate-backends.md) for various target languages
 
 ## TL;DR
-* [Install PureScript](http://www.purescript.org/download/)
-* [Install Node.js](https://nodejs.org/): `brew install node`/`pacman -S nodejs`/`choco install nodejs`/[Download Node.js](https://nodejs.org/)
-* [Install Pulp](https://github.com/bodil/pulp): `npm install -g pulp bower`
-* GO: `pulp init`, `bower install --save purescript-tuples`, `pulp build --to output.js`, etc.
+
+- [Install PureScript](http://www.purescript.org/download/)
```
Comment (hdgarrood) [inline README.md]: Maybe let's just remove everything under the TL;DR heading? The getting started guide didn't exist when this was written; now that it does I'm not sure this stuff should stay here. If people don't know this stuff already we should direct them to the getting started guide, because it's better maintained and easier to find.

**Comment by soupi** [comment]
Made the changes as requested.

> The only thing is I can't comment on the build tools stuff for editors other than Vim, so it would be good to get people who use each of those editors to look over those bits.

Yeah, I'd appreciate it as well.

**Comment by paf31** [comment]
👍 I like it!

**Comment by paf31** [comment]
Oh also - I think maybe we should test whether the Nix article is still up-to-date.

**Comment by soupi** [comment]
btw I included the Nix article because it's already in the current README, I have no problem removing it if that's what you prefer but I currently don't have the time to check if it still works or not. sorry.

**Comment by kRITZCREEK** [comment]
Thanks!
